opensrp-tasking/src/test/java/org/smartregister/tasking/view/TaskingHomeActivityTest.java

```diff
@@ -0,0 +1,865 @@
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.spray_card_view).getVisibility());
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.mosquito_collection_card_view).getVisibility());
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.larval_breeding_card_view).getVisibility());
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.potential_area_of_transmission_card_view).getVisibility());
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.indicators_card_view).getVisibility());
+//        assertEquals(GONE, taskingHomeActivity.findViewById(R.id.irs_verification_card_view).getVisibility());
+//    }
+
+    @Test
+    public void testPositionMyLocation() {
+        taskingHomeActivity = spy(taskingHomeActivity);
+        taskingHomeActivity.positionMyLocationAndLayerSwitcher();
+        FrameLayout.LayoutParams layoutParams = (FrameLayout.LayoutParams) myLocationButton.getLayoutParams();
+        assertEquals(0, layoutParams.topMargin);
+        assertEquals(30, layoutParams.bottomMargin);
+        assertEquals(Gravity.BOTTOM | Gravity.END, layoutParams.gravity);
+    }
+
+    //
+//    @Test
+//    public void testPositionMyLocationZambia() {
+//        taskingHomeActivity = spy(taskingHomeActivity);
+//        when(taskingHomeActivity.getBuildCountry()).thenReturn(Country.ZAMBIA);
+//        Whitebox.setInternalState(taskingHomeActivity, "myLocationButton", myLocationButton);
+//        taskingHomeActivity.positionMyLocationAndLayerSwitcher();
+//        FrameLayout.LayoutParams layoutParams = (FrameLayout.LayoutParams) myLocationButton.getLayoutParams();
+//        assertEquals(0, layoutParams.topMargin);
+//        int progressHeight = context.getResources().getDimensionPixelSize(R.dimen.progress_height);
+//        assertEquals(progressHeight + 40, layoutParams.bottomMargin);
+//        assertEquals(Gravity.BOTTOM | Gravity.END, layoutParams.gravity);
+//    }
+//
+//
+    @Test
+    public void testOpenFilterTaskActivity() {
+        taskingHomeActivity.findViewById(R.id.filter_tasks_fab).performClick();
+        verify(taskingLibraryConfiguration).openFilterTaskActivity(isNull(), any(TaskingHomeActivity.class));
+    }
+
+
+    @Test
+    public void testOpenTaskRegister() {
+        taskingHomeActivity.findViewById(R.id.task_register).performClick();
+        verify(taskingLibraryConfiguration).openTaskRegister(isNull(), any(TaskingHomeActivity.class));
+    }
+
+
+    @Test
+    public void testOnAddStructure() {
+        Whitebox.setInternalState(taskingHomeActivity, "taskingHomePresenter", taskingHomePresenter);
+        Whitebox.setInternalState(taskingHomeActivity, "mapHelper", taskingMapHelper);
+        taskingHomeActivity.findViewById(R.id.btn_add_structure).performClick();
+        verify(taskingHomePresenter).onAddStructureClicked(false);
+        verify(taskingMapHelper).isMyLocationComponentActive(any(), any());
+    }
+
+//    @Test
```

Comment:
I am not sure of the intention when commenting these out. Could you do something about it? Create an issue, todo, remove, fix